lib/mayaUsd/ufe/UsdUndoMaterialCommands.h

```diff
@@ -116,8 +126,8 @@ class MAYAUSD_CORE_PUBLIC UsdUndoAssignNewMaterialCommand : public Ufe::InsertCh
     void connectShaderToMaterial(Ufe::SceneItem::Ptr shaderItem, PXR_NS::UsdPrim materialPrim);
     void markAsFailed();
 
-    const Ufe::Path   _parentPath;
-    const std::string _nodeId;
+    std::map<pxr::UsdStageWeakPtr, std::vector<const Ufe::Path>> _stagesAndPaths;
```

Comment:
Sorry I didn't catch this earlier. We shouldn't use the namespace name directly, but instead the define. So "pxr::" should be "PXR_NS::" (see line 126).

Reply:
Once this and the merge conflict is fixed, you can run another preflight and if it passes add the label "ready for merge" and I'll merge it. I confirmed with our designer that the multiple selection workflow is okay.